apps/system/test/marionette/pinning_the_web_test.js

```diff
@@ -100,4 +99,36 @@ marionette('Pinning the Web', function() {
     assert(classes.indexOf('collapsible') < 0);
   });
 
+  suite.skip('Unpin site', function() {
```

Comment:
Why is this skipped? In case is because is using the doorhanger, can we change it to test the Pin Dialog unpin feature?